### PR TITLE
Align recent changes with Swagger

### PR DESCRIPTION
**Pull request by tomersh26**

### CODE REVIEW

**Review by erich-wang — CHANGES_REQUESTED** [review]
@tomersh26 , could you please:
1. Provide the swagger spec PR link related to 
2. Follow the [process](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec) to regenerate source code

**Comment by hvermis** [comment]
@erich-wang swagger PR-s
1. https://github.com/Azure/azure-rest-api-specs/pull/6717
2. https://github.com/Azure/azure-rest-api-specs/pull/6617

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 ## Current version
 ###  Feature Additions
 
+## Version 4.1.1
+###  Feature Additions
+- Add PostgreSql Sink
+- Add Partition settings for Oracle, SapTable
+- Add dedicated source and dataset for several relational sources including ODBC, MySql
+- Add Azure Data Explorer Command activity, which allows users executing Kusto Control Commands
+### Breaking Changes
```
Comment (hvermis) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: This is not a breaking change - you are adding new parameters to the constructor that have default values at the end, so  it won't break existing customers. Remove this line.

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 2]
```diff
@@ -3,6 +3,15 @@
 ## Current version
 ###  Feature Additions
 
+## Version 4.1.1
+###  Feature Additions
```
Comment (hvermis) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: Copy this whole section to csproj file for Datafactory, and update the version there to 4.1.1.
Also update AssemblyFileVersion to 4.1.1.0 in Properties

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/SdkInfo_DataFactoryManagementClient.cs [hunk 3]
```diff
@@ -1,40 +0,0 @@
-
```
Comment (hvermis) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/SdkInfo_DataFactoryManagementClient.cs]: please do not remove this file

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md (hunk 1 above)
Comment (tomersh26) [reply sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: This PR contains both my change and Wenbo's changes. The note under the Breaking Changes is Wenbo's note which is related to her changes in PostgreSql.
You can have a look at her original PR which I took and integrated into my PR: https://github.com/Azure/azure-sdk-for-net/pull/7012/commits/10fec6da7d8843aad9a9fc9ba0bcf48a889e4b34.

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md (hunk 2 above)
Comment (tomersh26) [reply sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: I guess you meant changing both the AssemblyFileVersion and the AssemblyVersion under the AssemblyInfo.cs file. I changed them both to 4.1.1.0.

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/SdkInfo_DataFactoryManagementClient.cs (hunk 3 above)
Comment (tomersh26) [reply sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/SdkInfo_DataFactoryManagementClient.cs]: Ok.
Just note that it was removed by autorest (both for me and for Wenbo).

**Comment by tomersh26** [comment]
> @tomersh26 , could you please:
> 
> 1. Provide the swagger spec PR link related to
> 2. Follow the [process](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec) to regenerate source code

@erich-wang - It seems that i don't have permissions to that link. I'm getting an error 400 - Page not found.

**Comment by erich-wang** [comment]
@tomersh26, are you from Microsoft? If yes, you should first link your github account with microsoft email and join Azure organization, then you should be able to see the link since it is in private repo.

> > @tomersh26 , could you please:
> > 
> > 1. Provide the swagger spec PR link related to
> > 2. Follow the [process](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec) to regenerate source code
> 
> @erich-wang - It seems that i don't have permissions to that link. I'm getting an error 400 - Page not found. In general, I've followed the process as I was guided by ADMS folks. I used autorest providing a path to the recent changes for the public swagger, and wrote all the required tests.

**Comment by erich-wang** [comment]
@tomersh26, I did not see **eng\mgmt\mgmtmetadata\<rpName>.txt** which should be in PR if you follow the latest process.

**Review by erich-wang — APPROVED** [review]

**Pull request merged**